docs/guide/search/solr.md

```diff
@@ -4,15 +4,15 @@
 
 ## How to set up Solr search engine
 
-!!! note "Enable the bundle"
+!!! note "Installing the bundle"
 
-    If you have previously disabled the bundle, add/update composer dependencies:
+    If you have previously removed the bundle, add/update composer dependencies:
     ``` bash
-    composer require --no-update ezsystems/ezplatform-solr-search-engine:~2.0
+    composer require --no-update ezsystems/ezplatform-solr-search-engine:~3.0
     composer update
     ```
-
-    Make sure `EzPublishSolrSearchEngineBundle` is activated with the following line in the `app/AppKernel.php` file: `new EzSystems\EzPlatformSolrSearchEngineBundle\EzSystemsEzPlatformSolrSearchEngineBundle()`
+    
+    Symfony Flex will enable the bundle for you when installing the package.
```

Comment:
@dabrt FYI reverted part of this for 2.5 in 7cabc6801085a1ef16ac2a24e1c274648cf8e100, I forgot to mention the changes like this was meant for 3.x only.